README.md

```diff
@@ -18,14 +18,20 @@ cp node/.env.template node/.env
 
 5. Paste your HubSpot API Key as the value for HUBSPOT_API_KEY in .env
 
-6. Follow the language instructions on how to run. For example, if you want to run the Node server:
+6. Follow the language instructions on how to run.
 
-```
+```Node
 cd node # there's a README in this folder with instructions
 npm install
 ./bin/cli.js
 ```
 
+```Php
```

Comment:
I don't think it makes sense to provide setup instructions for all languages in top-level README. lets leave only `node.js` as an example.